src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -171,6 +171,10 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
     this.selectedTechnologies = $event;
   }
 
+  getActualDate(): string {
```

Comment:
I think you should change this to `getCurrentDate`. And update all references from `actual date` to `current date`